haystack/nodes/audio/answer_to_speech.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from typing import Any, Union, Callable, Optional, List, Dict, Tuple
+
+import hashlib
+from pathlib import Path
+
+from haystack.nodes import BaseComponent
+from haystack.schema import Answer, AudioAnswer, GeneratedAudioAnswer
+from haystack.nodes.audio._text_to_speech import TextToSpeech
+
+
+class AnswerToSpeech(BaseComponent):
+    """
+    This node converts text-based Answers into AudioAnswers, where the answer and its context are
+    read out into an audio file.
+    """
+
+    outgoing_edges = 1
+
+    def __init__(
+        self,
+        model_name_or_path: Union[str, Path] = "espnet/kan-bayashi_ljspeech_vits",
+        generated_audio_dir: Path = Path(__file__).parent / "generated_audio_answers",
+        audio_format: str = "wav",
+        subtype: str = "PCM_16",
+        audio_naming_function: Callable = lambda text: hashlib.md5(text.encode("utf-8")).hexdigest(),
+        transformers_params: Optional[Dict[str, Any]] = None,
+    ):
+        """
+        Convert an input Answer into an audio file containing the answer's answer and context read out loud.
+
+        :param model_name_or_path: the text to speech model, for example `espnet/kan-bayashi_ljspeech_vits`
+        :param generated_audio_dir: folder to save the audio file to
+        :param audio_format: the format to save the audio into (wav, mp3, ...)
+        :param subtype: see soundfile.write()
```

Comment:
I would prefer if we copy over the information from `soundfile.write()` rather than referring to it. Much easier for users to understand what value to set and how it influences the result.

Reply:
Unfortunately I just noticed that `soundfile.write()` also has almost no documentation on the matter :see_no_evil: In addition, the subtype seems to be dependent on the uncompressed format selected. I will try to find a better explanation of these strings somewhere and add it here